### PR TITLE
Add ADIS IMUs

### LINKED ISSUE #240: Add ADIS Gyro Support
Currently not supported in generator setup

### PR DESCRIPTION
**Pull request by Piphi5**
Fixes #240

### CODE REVIEW

**Comment by calcmogul** [comment]
The development build that includes the ADIS classes finally published, so this PR can start using them.

File: README.md [hunk 1]
```diff
@@ -16,16 +16,12 @@ This is a list of the vendors that are currently supported by SysId for 2022:
 
 ### Hardware Supported by WPILib
 
+- ADIS Gyro
```
Comment (calcmogul) [inline README.md:19]:
```suggestion
- ADIS16448 IMU
- ADIS16470 IMU
```

File: README.md [hunk 2]
```diff
@@ -16,16 +16,12 @@ This is a list of the vendors that are currently supported by SysId for 2022:
...
 - Analog Gyro
```
Comment (calcmogul) [inline README.md:20]: Are these in any particular order?

**Review by calcmogul — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Piphi5) [reply README.md:20]: Not really, should we alphabetize it?

Comment (calcmogul) [reply README.md:20]: Yea

File: sysid-library/src/main/cpp/generation/SysIdSetup.cpp [hunk 3]
```diff
@@ -264,13 +264,37 @@ void SetupEncoders(
...
+    fmt::print("Setup SPI kOnboardCS2");
+    return frc::SPI::Port::kOnboardCS2;
+  } else if (name == "kOnboardCS3") {
+    fmt::print("Setup SPI kOnboardCS3");
+    return frc::SPI::Port::kOnboardCS3;
+  } else if (name == "SPI.kMXP") {
+    fmt::print("Setup SPI SPI.kMXP");
+    return frc::SPI::Port::kMXP;
+  } else {
+    throw std::runtime_error(fmt::format("{} is not a supported SPI Port\n"));
```
Comment (calcmogul) [inline sysid-library/src/main/cpp/generation/SysIdSetup.cpp:284]: You forgot to put the name as a format argument.

**Review by calcmogul — CHANGES_REQUESTED** [review]

**Review by calcmogul — APPROVED** [review]

**Pull request merged**